test/e2e/tests/ipfs-toggle.spec.js

```diff
@@ -0,0 +1,73 @@
+const { strict: assert } = require('assert');
+const { withFixtures, convertToHexValue } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Settings', function () {
+  const smartContract = SMART_CONTRACTS.ERC1155;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
```

Comment:
you can use `defaultGanachOptions`

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/20360/commits/ea179ad09018fc212e9eb0a27cab98992bda3895